bigquery/tests/system.py

```diff
@@ -1005,6 +1006,10 @@ def _generate_standard_sql_types_examples(self):
                 'sql': 'SELECT TIME(TIMESTAMP "%s")' % (stamp,),
                 'expected': naive.time(),
             },
+            {
+                'sql': 'SELECT NUMERIC "123456789.123456789"',
+                'expected': decimal.Decimal('123456789.123456789'),
+            },
```

Comment:
Quibble:  it would be better to name the string literal constant, and format it into the query (as with the timestamp above).

Reply:
Agreed, done.